### DISCUSSION TITLE
Where I will find the jar file for grpc-web java?

### DISCUSSION THREAD

**Discussion by saumyaprakash30** [grpc/grpc-web#1190] [Q&A - General]
I am not able to find the jar file which will be used for gRPC-web java. I also want to know Is io.grpcweb will be available in that JAR file. This [page](https://github.com/grpc/grpc-web/blob/master/src/connector/README.md#:~:text=Rebuild%20your%20Service%20with%20the%20Java%20jar%20file%20provided%20by%20this%20project%20and%20re%2Ddeploy) says that jar file is provided with project.

**Comment by sampajano** [comment]
Thanks for the question! But sorry unfortunately that the Java proxy project is ON HOLD right now and is not being actively supported at this moment..

You could try building it from the java source files if you want to? (sorry again that i couldn't help :))
